main.go

```diff
@@ -119,6 +121,16 @@ func getNamespace() (string, error) {
 	return ns, nil
 }
 
+// The name of the secret containing the addon params
+func getConfigName() string {
+
+	configName, found := os.LookupEnv(configSecretEnvVar)
+	if !found {
+		configName = controllers.AddonSecretName
+	}
+	return configName
+}
+
```

Comment:
This is the wrong place to do it as the value should be read from the `managedOCS` CR (with a default value, of course)

Reply:
It was decided to use an env value to store this instead.

Reply:
This is correct, we decided to have 2 secrets, the first one will point at the secret, the second one will be used to get the storage unit per OSD